Subsurface/Node/Common/peripherals/src/I2CInterface.cpp

```diff
@@ -0,0 +1,42 @@
+#include "I2CInterface.h"
+
+I2CInterface::I2CInterface(I2C_HandleTypeDef *hi2c):m_hi2c(hi2c) {}
```

Comment:
I think this looks better like:
```C
I2CInterface::I2CInterface(I2C_HandleTypeDef *hi2c)
    : m_hi2c(hi2c)
{}
```
wdyt?

Reply:
I didn't even realize i had it done the other way, there was a bunch of weird stuff going on when I was switching between clion and vs code and i think I have different formatting rules on vs code (for some reason). I'm a big advocate for initializer lists being on a separate line (I'll get this sorted to out for the future)